sdk/extensions/Microsoft.Extensions.Azure/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 ### Features Added
 
+- Added support for [federated managed identity](https://learn.microsoft.com/entra/workload-id/workload-identity-federation-config-app-trust-managed-identity?tabs=microsoft-entra-admin-center#azureidentity) support in the client factory by specifying configuration item `credential` as "managedidentityasfederatedidentity" and providing the following named configuration items:
+
+  - `tenantId` : The tenant where the target resource was created
+  - `clientId` : The client identifier for the application, which must be granted access on the target resource
+  - `managedIdentityClientId` : The managed identity which you configured as a Federated Identity Credential (FIC)
```

Comment:
Should we support (mutually exclusively) resourceId and objectId also?

Reply:
Same answer as above.  😄

Reply:
Yes, we should because those are the 2 other IDs that can be used with user-assigned MI.

Reply:
Done.